src/Symfony/Component/HttpKernel/Tests/HttpKernelTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\HttpKernel\Tests;
 
+use Error;
```

Comment:
same as `\Throwable`, please inline `\Error` where needed

Reply:
Heck. my storm automatically does it :D done.